sites/svelte-5-preview/src/routes/docs/content/01-api/02-runes.md

```diff
@@ -64,6 +64,28 @@ Objects and arrays [are made reactive](/#H4sIAAAAAAAAE42QwWrDMBBEf2URhUhUNEl7c21
 
 In non-runes mode, a `let` declaration is treated as reactive state if it is updated at some point. Unlike `$state(...)`, which works anywhere in your app, `let` only behaves this way at the top level of a component.
 
+## `$state.frozen`
+
+Similar to `$state`, `$state.frozen` is also declared and can be used in many of the same ways (including on classes). However, the properties of any object and arrays are treated as read-only, and cannot be mutated. So if you intend to use objects as state and you want to mutate their properties and have reactivity work by default, it's recommended you use `$state` instead.
+
+For the cases where you don't want Svelte's reactivity to apply deeply to state, and for those who might want to have more control over their data structures, you might find `$state.frozen` useful. Furthermore, `$state.frozen` is ideal for those who want to work with data using immutable patterns rather than mutable patterns.
```

Comment:
It's not really clear to me from reading this whether freezing an object will have some effects apply to it on a deep or shallow basis. I do see the note in the callout below which is quite helpful, but I wonder after having read this what happens if someone tries to do `a.b.c = 'newValue';`

I might remove "and for those who might want to have more control over their data structures" because I think it could be interpreted multiple ways. Like if someone passes you frozen state maybe you have less control over it